stan/math/mix/functor/laplace_marginal_density.hpp

```diff
@@ -38,13 +39,14 @@ struct laplace_options_base {
    * using an LU decomposition (more general, but slower)
    */
   int solver{1};
-  /* Maximum number of steps in line search */
-  int max_steps_line_search{0};
-  /* iterations end when difference in objective function is less than tolerance
+  /**
+   * iterations end when difference in objective function is less than tolerance
+   * Default is sqrt(machine_epsilon)
```

Comment:
That's very interesting... Did you find that reducing the tolerance (from 1e-6 to sqrt(machine_epsilon)) and increasing max step size improves performance on difficult problems (i.e. the roach problem)?